### PR TITLE
Added Org Readme

### LINKED ISSUE #40: Add a organization README
Organizations can have READMEs now. This would help people navigate the large (and growing) OpenSearch org.

Example:
https://github.com/twitter

Screen Shot 2021-09-17 at 10 31 08 AM

It comes from:
.github/profile/README.md

### PR DESCRIPTION
**Pull request by dtaivpp**
Signed-off-by: David Tippett <dtip@amazon.com>

### Description
Adds an organizational readme that briefly describes OpenSearch and how to get involved. 

### Issues Resolved
Closes #40 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dblock — APPROVED** [review]
I like it!

**Review by dblock — CHANGES_REQUESTED** [review]
Let's rename banner.jpg to lowercase banner.jpg, to help case-sensitive systems.

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+OpenSearch is a community-driven, Apache 2.0-licensed open source search and analytics suite that makes it easy to ingest, search, visualize, and analyze data. Developers build with OpenSearch for use cases such as application search, log analytics, data observability, data ingestion, and more.
+
+## Contributing ✍️
+
+We are built 🧱 by the community for the community. If you would like to contribute there are many ways: 
+
+- 👀 Check out a project's `CONTRIBUTING.md` file to learn how to contribute. 
+- [Blogs](https://github.com/opensearch-project/project-website) and [Documentation Updates](https://github.com/opensearch-project/documentation-website)!
+- Finally, make sure to try [OpenSearch](https://opensearch.org/docs/latest/opensearch/install/docker/) 🔎, [OpenSearch Dashboards](https://playground.opensearch.org/app/home) 🖥, and our Plugins/Client Libraries 📚 and leave feedback! 
+
```
Comment (dblock) [inline profile/README.md:12]: Remove double-space.

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+## Get Involved in our Community!
+
+There are several places where our community meets make sure to check them out!
+
+- 📝 [Forum](https://forum.opensearch.org/)
+- ▶️ [YouTube](https://www.youtube.com/c/OpenSearchProject)
+- 🐤 [Twitter](https://twitter.com/OpenSearchProj)
+- 🧑‍💼 [LinkedIn](https://www.linkedin.com/company/opensearch-project/)
+
```
Comment (dblock) [inline profile/README.md:22]: Remove double-space?

Comment (dtaivpp) [reply profile/README.md:21]: Resolved, its a personal preference but not necessary.

**Review by peternied — APPROVED** [review]
Nicely done!

**Review by dblock — APPROVED** [review]

**Comment by dblock** [comment]
Merged. Looks like the banner will need an absolute URL, will you please fix @dtaivpp?

**Comment by dtaivpp** [comment]
@dblock I'll get that updated

**Pull request merged**